website/client/components/guilds/index.vue

```diff
@@ -0,0 +1,32 @@
+<template lang="pug">
+.row
+  secondary-menu.col-12
+    router-link.nav-link(:to="{name: 'tavern'}", exact, :class="{'active': $route.name === 'tavern'}") {{ $t('tavern') }}
+    router-link.nav-link(:to="{name: 'myGuilds'}", :class="{'active': $route.name === 'myGuilds'}") {{ $t('myGuilds') }}
+    router-link.nav-link(:to="{name: 'guildsDiscovery'}", :class="{'active': $route.name === 'guildsDiscovery'}") {{ $t('guildsDiscovery') }}
+
+  .col-12
+    router-view
+
+  group-form-modal
+</template>
+<!-- .col-md-2
```

Comment:
Yes, but the subnav doesn't allow for this. However, I think the designers moved this button.